modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/app-banner/configuration.json

```
@@ -0,0 +1,20 @@
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"description": "johndoe@example.com",
+					"label": "Publisher Email",
+					"name": "publisherEmail",
+					"type": "text"
+				},
+				{
+					"description": "11111",
+					"label": "Product ID",
+					"name": "productID",
+					"type": "text"
+				}
```

The name of this property would be camel case.
```suggestion
				{
					"description": "11111",
					"label": "Product ID",
					"name": "productId",
					"type": "text"
				}
```